src/screens/SearchScreen/components/DictionaryEntryExpansion.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import Expression from './Expression';
+
+const DictionaryEntryExpansion = ({expressions}) => {
+  return (
+    <View>
+      {expressions[0]
+        ? expressions.map(
+            expression =>
+              (expression = (
```

Comment:
We looked at this pretty closely and discussed why this is unnecessary and a bit confusing. Just to summarize: `Array.map` yields a new array populated with the values returned from the lambda (so no assignment necessary).